packages/reports/src/Containers/open-positions.tsx

```diff
@@ -190,23 +191,23 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
     const { isDesktop } = useDevice();
     const previous_active_positions = usePrevious(active_positions);
     const contract_types = [
-        { text: localize('Options'), is_default: !is_multiplier && !is_accumulator },
-        { text: localize('Multipliers'), is_default: is_multiplier },
-        { text: localize('Accumulators'), is_default: is_accumulator },
+        { text: 'Options', is_default: !is_multiplier && !is_accumulator },
+        { text: 'Multipliers', is_default: is_multiplier },
+        { text: 'Accumulators', is_default: is_accumulator },
     ];
     const [contract_type_value, setContractTypeValue] = React.useState(
-        contract_types.find(type => type.is_default)?.text || localize('Options')
+        contract_types.find(type => type.is_default)?.text || 'Options'
     );
     const prev_contract_type_value = usePrevious(contract_type_value);
-    const accumulator_rates = [localize('All growth rates'), '1%', '2%', '3%', '4%', '5%'];
+    const accumulator_rates = ['All growth rates', '1%', '2%', '3%', '4%', '5%'];
     const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
     const prev_accumulator_rate = usePrevious(accumulator_rate);
     const is_accumulator_selected = contract_type_value === contract_types[2].text;
     const is_multiplier_selected = contract_type_value === contract_types[1].text;
     const contract_types_list = contract_types
-        .filter(contract_type => contract_type.text !== localize('Accumulators') || !hide_accu_in_dropdown)
-        .map(({ text }) => ({ text, value: text }));
-    const accumulators_rates_list = accumulator_rates.map(value => ({ text: value, value }));
+        .filter(contract_type => contract_type.text !== 'Accumulators' || !hide_accu_in_dropdown)
```

Comment:
Hmm, please feel free to remove the label comparison here and utilize keys or any static values instead, if you want. It's not a good practise here to compare localized labels, but it's not your change though.

Reply:
I also don't think that we should compare 2 localized labels, so I removed  it. Add localize in the string below, just before we pass it to component